src/ipc/bridge.cc

```diff
@@ -736,10 +736,11 @@ static void initRouterTable (Router *router) {
     );
   });
 
-  /**
-   * Creates a directory at `path` with an optional mode.
+ /**
```

Comment:
now there is one more space needed :)
it should look like this (the first `*` in `/**` should be on the same position as the following `*`s)
```
/**
 *
 */
```

Reply:
Fixed.